streamalert/scheduled_queries/query_packs/configuration.py

```diff
@@ -112,5 +110,6 @@ def register(cls, config):
         cls.QUERY_PACKS[name] = config
 
     @classmethod
-    def load_packs(cls):
-        import_folders(PACKS_DIRECTORY)
+    def load_packs(cls, directories):
+        for dir in directories:
```

Comment:
`import_folders` takes an arbitrary # of args (using `*args`).. all you have to do here is use list expansion:

```python
def load_packs(cls, directories):
    import_folders(*directories)
```